### ISSUE TITLE
Client does not start

### ISSUE THREAD

**Issue by augjoh** [polhenarejos/acme_email#5]
When starting the client on a fresh installation, an exception is thrown:

```
$ python3 cli.py --help
Traceback (most recent call last):
  File "/builds/platynum/certification-authority/flows/acme_email/cli.py", line 8, in <module>
    from certbot._internal import configuration
ImportError: cannot import name 'configuration' from 'certbot._internal' (/usr/lib/python3.9/site-packages/certbot/_internal/__init__.py)
```

**Comment by polhenarejos** [comment]
Which version of certbot are you using?

```python3 -c 'import certbot;print(certbot.__version__)'```

**Comment by augjoh** [comment]
The one that is provided using `setup.py`:

```
$ pip install .
Processing /builds/platynum/certification-authority/flows/acme_email
Collecting certbot>=1.1.0
  Downloading certbot-1.18.0-py2.py3-none-any.whl (258 kB)
Collecting acme>=0.29.0
  Downloading acme-1.18.0-py2.py3-none-any.whl (43 kB)
Requirement already satisfied: setuptools in /usr/lib/python3.9/site-packages (from certbot-castle==0.8.3.dev0) (52.0.0)
Collecting zope.interface
  Downloading zope.interface-5.4.0.tar.gz (249 kB)
Collecting imapclient
  Downloading IMAPClient-2.2.0-py2.py3-none-any.whl (179 kB)
Requirement already satisfied: chardet in /usr/lib/python3.9/site-packages (from acme>=0.29.0->certbot-castle==0.8.3.dev0) (4.0.0)
Requirement already satisfied: cryptography>=2.1.4 in /usr/lib/python3.9/site-packages (from acme>=0.29.0->certbot-castle==0.8.3.dev0) (3.3.2)
Collecting josepy>=1.1.0
  Downloading josepy-1.8.0-py2.py3-none-any.whl (61 kB)
Collecting pyrfc3339
  Downloading pyRFC3339-1.1-py2.py3-none-any.whl (5.7 kB)
Collecting pytz
  Downloading pytz-2021.1-py2.py3-none-any.whl (510 kB)
Collecting PyOpenSSL>=17.3.0
  Downloading pyOpenSSL-20.0.1-py2.py3-none-any.whl (54 kB)
Collecting requests-toolbelt>=0.3.0
  Downloading requests_toolbelt-0.9.1-py2.py3-none-any.whl (54 kB)
Requirement already satisfied: requests>=2.14.2 in /usr/lib/python3.9/site-packages (from acme>=0.29.0->certbot-castle==0.8.3.dev0) (2.25.1)
Collecting zope.component
  Downloading zope.component-5.0.1-py2.py3-none-any.whl (68 kB)
Collecting parsedatetime>=2.4
  Downloading parsedatetime-2.6-py3-none-any.whl (42 kB)
Collecting ConfigArgParse>=0.9.3
  Downloading ConfigArgParse-1.5.2-py3-none-any.whl (20 kB)
Requirement already satisfied: distro>=1.0.1 in /usr/lib/python3.9/site-packages (from certbot>=1.1.0->certbot-castle==0.8.3.dev0) (1.5.0)
Collecting configobj>=5.0.6
  Downloading configobj-5.0.6.tar.gz (33 kB)
Requirement already satisfied: six in /usr/lib/python3.9/site-packages (from configobj>=5.0.6->certbot>=1.1.0->certbot-castle==0.8.3.dev0) (1.15.0)
Requirement already satisfied: cffi>=1.12 in /usr/lib/python3.9/site-packages (from cryptography>=2.1.4->acme>=0.29.0->certbot-castle==0.8.3.dev0) (1.14.5)
Requirement already satisfied: pycparser in /usr/lib/python3.9/site-packages (from cffi>=1.12->cryptography>=2.1.4->acme>=0.29.0->certbot-castle==0.8.3.dev0) (2.20)
Requirement already satisfied: idna<3.3,>=2.5 in /usr/lib/python3.9/site-packages (from requests>=2.14.2->acme>=0.29.0->certbot-castle==0.8.3.dev0) (3.2)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/lib/python3.9/site-packages (from requests>=2.14.2->acme>=0.29.0->certbot-castle==0.8.3.dev0) (1.26.5)
Requirement already satisfied: certifi>=2017.4.17 in /usr/lib/python3.9/site-packages (from requests>=2.14.2->acme>=0.29.0->certbot-castle==0.8.3.dev0) (2020.12.5)
Collecting zope.hookable>=4.2.0
  Downloading zope.hookable-5.1.0.tar.gz (21 kB)
Collecting zope.event
  Downloading zope.event-4.5.0-py2.py3-none-any.whl (6.8 kB)
Using legacy 'setup.py install' for certbot-castle, since package 'wheel' is not installed.
Using legacy 'setup.py install' for configobj, since package 'wheel' is not installed.
Using legacy 'setup.py install' for zope.hookable, since package 'wheel' is not installed.
Using legacy 'setup.py install' for zope.interface, since package 'wheel' is not installed.
Installing collected packages: pytz, PyOpenSSL, zope.interface, zope.hookable, zope.event, requests-toolbelt, pyrfc3339, josepy, zope.component, parsedatetime, configobj, ConfigArgParse, acme, imapclient, certbot, certbot-castle
    Running setup.py install for zope.interface: started
    Running setup.py install for zope.interface: finished with status 'done'
    Running setup.py install for zope.hookable: started
    Running setup.py install for zope.hookable: finished with status 'done'
    Running setup.py install for configobj: started
    Running setup.py install for configobj: finished with status 'done'
    Running setup.py install for certbot-castle: started
    Running setup.py install for certbot-castle: finished with status 'done'
Successfully installed ConfigArgParse-1.5.2 PyOpenSSL-20.0.1 acme-1.18.0 certbot-1.18.0 certbot-castle-0.8.3.dev0 configobj-5.0.6 imapclient-2.2.0 josepy-1.8.0 parsedatetime-2.6 pyrfc3339-1.1 pytz-2021.1 requests-toolbelt-0.9.1 zope.component-5.0.1 zope.event-4.5.0 zope.hookable-5.1.0 zope.interface-5.4.0
```

**Comment by polhenarejos** [comment]
Certbot was upgraded to 1.18, modifying some core procedures.

Should be fixed in 715c892.
